main.go

```diff
@@ -2,9 +2,18 @@ package main
 
 import (
 	"os"
+	"fmt"
 )
 
+var vetError = fmt.Sprintf("", 1, 2, 3)
+
 func main() {
 	c := &CLI{OutStream: os.Stdout, ErrStream: os.Stderr}
 	os.Exit(c.Run(os.Args))
 }
+
+func Hoge(){
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function Hoge should have comment or be unexported